base/show.jl

```diff
@@ -982,7 +982,7 @@ end
 function show_call(io::IO, head, func, func_args, indent)
     op, cl = expr_calls[head]
     if (isa(func, Symbol) && func !== :(:) && !(head === :. && isoperator(func))) ||
-            (isa(func, Expr) && (func.head == :. || func.head == :curly)) ||
+            (isa(func, Expr) && (func.head == :. || func.head == :curly || func.head == :macroname)) ||
```

Comment:
`:macrocall`?

Reply:
Ah, I see, this is something you added.

Reply:
Yes, it was either that or special case the printing of `Symbol` in a couple of places.

Reply:
But the special case would require carrying around / somehow detecting extra context so this seemed like a simple way to do that.